src/lambdas/upload_file.py

```diff
@@ -6,6 +6,24 @@
 import boto3
 from dynamodb_json import json_util as d_json
 
+class FileData():
+    def __init__(self, name: str, type: str, desc: str, tags: List[str], size: int, upload_date: datetime, last_modified: datetime, creation_date: datetime):
+        self.name = name
+        self.type = type
+        self.desc = desc
+        self.tags = tags[:]
+        self.upload_date = upload_date
+        self.last_modified = last_modified
+        self.creation_date = creation_date
+        self.size = size
```

Comment:
Consider using the `@dataclass` decorator. Would allow declaring member variables in typical oop fashion and make the `__init__` and `__str__` methods redundant.

Reply:
Done in [c213019](https://github.com/magley/klau-drive/pull/7/commits/c213019e33f3f37a6a18d9665f169316d5c43bbc). It's not as elegant as I thought it would be.

Reply:
Only now have I noticed that you meant the `FileData` class and not the GUI classes. [c9bd94a](https://github.com/magley/klau-drive/pull/7/commits/c9bd94ae63023ccd75f23c26164b2fd79b915c4b)